components/modal-card/index.js

```diff
@@ -273,6 +273,20 @@ export default function ModalCard({ children }) {
                             </button>
                           </div>
                         </div>
+                        {signer && (
```

Comment:
beautiful. I'd just remove the signer && condition from here. Other than that, this is perfect. I really like where you placed the disclaimer in the faq page. Amazing work. Thank you

Reply:
ok. done.